app/src/cambodia/java/org/eyeseetea/malariacare/strategies/BaseActivityStrategy.java

```diff
@@ -19,16 +36,74 @@ public void onStop() {
     @Override
     public void onCreate() {
 
+        mAuthenticationManager = new AuthenticationManager(mBaseActivity);
+        mLogoutUseCase = new LogoutUseCase(mAuthenticationManager);
+        mLoginUseCase = new LoginUseCase(mAuthenticationManager);
     }
 
     @Override
     public void onCreateOptionsMenu(Menu menu) {
 
     }
 
+
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        return false;
+        int id = item.getItemId();
+
+        switch (id) {
+            case R.id.demo_mode:
+                runDemoMode();
+                break;
+            default:
+                return false;
+        }
+        return true;
+    }
+
+    public void runDemoMode() {
+        PopulateDB.wipeSurveys();
```

Comment:
@idelcano according to the clean architecture, presentation layer should not call directly to the data layer to clean the database, better invoke LogoutUseCase which already clean the database and org units from preferences